variables.tf

```diff
@@ -122,6 +122,18 @@ variable "network_type" {
     description = "Specify the name of the network adapter type to use for creating hosts"
 }
 
+variable "sriov_vnic_failover_adapter" {
+    # Eg: 1 = VNIC without failover; 2 = VNIC failover with 2 SR-IOV LPs
+    default = 1
```

Comment:
I read about it in API docs. Would recommend a validation block for `vnic_required_vfs cannot be greater than 6`

Reply:
Also good point to have a validation, so that a user cannot enter a higher number. Have to check what is possible with validators, but I would prefer a stop of the deployment with a message about the restriction.

Reply:
Just a question, because I'm not (yet) familiar with the process in GitHub: Will a change like this be a new PR, or is there a different process like squash commits?

Reply:
> 
> Also good point to have a validation, so that a user cannot enter a higher number. Have to check what is possible with validators, but I would prefer a stop of the deployment with a message about the restriction.

You could use [Terraform Custom Validation Rules ](https://www.terraform.io/docs/language/values/variables.html#custom-validation-rules) but I am okay with just a mention in the documentation.

Reply:
> 
> Just a question, because I'm not (yet) familiar with the process in GitHub: Will a change like this be a new PR, or is there a different process like squash commits?

Since this is a small change you could make the changes and `ammend` to the same commit and then force push to your branch.

Reply:
Ok. Is done. Added and tested validation rule, updated documentation and changed the variable name from sriov_vnic_failover_adapter to sriov_vnic_failover_vfs (because we don't define adapters here, we define virtual functions and the users might get mislead with the original name).